src/ensemble/ensemble_analysis.jl

```diff
@@ -140,12 +140,16 @@ function timeseries_point_meanvar(sim,ts)
   end
   DiffEqArray(means,ts),DiffEqArray(vars,ts)
 end
+
+timeseries_point_meancov(sim,ts) = timeseries_point_meancov(sim,ts[1:end-1],ts[2:end])
 function timeseries_point_meancov(sim,ts1,ts2)
   reshape([timepoint_meancov(sim,t1,t2) for t1 in ts1 for t2 in ts2],length(ts1),length(ts2))
 end
+timeseries_point_meancor(sim,ts) = timeseries_point_meancor(sim,ts[1:end-1],ts[2:end])
 function timeseries_point_meancor(sim,ts1,ts2)
   reshape([timepoint_meancor(sim,t1,t2) for t1 in ts1 for t2 in ts2],length(ts1),length(ts2))
 end
+timeseries_point_weighted_meancov(sim,ts) = timeseries_point_weighted_meancov(sim,ts[1:end-1],ts[2:end])
```

Comment:
I guess it should be
```suggestion
timeseries_point_weighted_meancov(sim,W,ts) = timeseries_point_weighted_meancov(sim,W,ts[1:end-1],ts[2:end])
```
?